serverless.yml

```diff
@@ -86,6 +87,14 @@ functions:
     environment:
       DEFAULT_DYNAMODB_TABLE: ${self:custom.DYNAMO_DB_TABLE}
       DEFAULT_EXPIRATION_PERIOD: ${self:custom.DEFAULT_EXPIRATION_PERIOD}
+      DEFAULT_REGION: ${self:custom.DEFAULT_REGION}
+
+  gitHook:
+    handler: src/handlers/gitHook.index
+    events:
+      - http:
+          path: /index
```

Comment:
Can we give this a more descriptive path? Since this is a POST it should probably be a verb (action). e.g. /deleteStack /mergeBranch

Reply:
done